examples/contrib/oed/models/bayes_linear.py

```diff
@@ -1,15 +1,126 @@
 import warnings
+from collections import OrderedDict  # noqa: F401  # for docstrings
+from functools import partial
 import torch
 from torch.nn.functional import softplus
-import numpy as np
+from torch.distributions import constraints
+from torch.distributions.transforms import AffineTransform, SigmoidTransform
 
 import pyro
 import pyro.distributions as dist
+from pyro.contrib.oed.util import rmv
 
 
-def bayesian_linear_model(design, w_means=None, w_sqrtlambdas=None, obs_sd=None,
+def known_covariance_linear_model(coef_mean, coef_sd, observation_sd,
+                                  coef_label="w", observation_label="y"):
+    model = partial(bayesian_linear_model,
+                    w_means={coef_label: coef_mean},
+                    w_sqrtlambdas={coef_label: 1./(observation_sd*coef_sd)},
+                    obs_sd=observation_sd,
+                    response_label=observation_label)
+    # For testing, add these
+    model.obs_sd = observation_sd
+    model.w_sds = {coef_label: coef_sd}
+    return model
```

Comment:
These are just utility functions calling `bayesian_linear_model` with different combinations of arguments